spydra/src/main/java/com/spotify/spydra/submitter/api/DynamicSubmitter.java

```diff
@@ -64,6 +64,10 @@ public DynamicSubmitter() {
     this(new DataprocAPI(), new GcpUtils());
   }
 
+  public DynamicSubmitter(String account) {
+    this(new DataprocAPI(account), new GcpUtils());
```

Comment:
You're missing to forward account to https://github.com/spotify/spydra/pull/40/files#diff-f65deef2eaa5e91f33cbd8793e2e83a2R71